applications/machine_learning/README.rst

```diff
@@ -275,6 +275,28 @@ By default, the application uses the following LED effects:
     * LED blinks with an average frequency if it is connected, but is not actively forwarding data.
     * LED blinks rapidly if it is connected and is actively forwarding data.
 
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+In the Machine Learning application specified modules can be suspended or power off there are not in use.
+
+* In the suspended state, the device maintains the active connection.
+* In the powered off state, the CPU is switched to the off mode.
+
+In both cases, most of the functionalities are disabled.
+For example, LEDs and sensors are turned off and advertising is stopped.
+
+Any button press can wake up the device.
+
+For the Thingy:53 sensors supports active power management.
+As long as device acceleration is detected the board is kept in active state.
+When the board goes into sleep it can also be woken up by acceleration threshold - it means by moving the device.
+
+You can define the amount of time after which the peripherals are suspended or powered off in :kconfig:`CONFIG_CAF_POWER_MANAGER_TIMEOUT`.
```

Comment:
Actually Thingy:53 sets this to 10 seconds.

Reply:
Well.... it does not change the defaults. Moreover, after #6272 this settings would be reverted to default.

Reply:
We can leave it as it is if this will be reverted to 120 seconds.